README.md

```diff
@@ -131,9 +131,26 @@ In case using Vagrant is not desired, ansible playbooks and roles from this repo
 on test.example.com machine, where the dev env should be deployed
 ```sh
 useradd vagrant
+echo vagrant:redhat | chpasswd
 echo "vagrant	ALL=(ALL)	NOPASSWD: ALL" >> /etc/sudoers.d/vagrant
 ```
 
+Here we can see the initial configuration, when creating the python virtual environment and preparing the setup (MACOS Users or Regular Linux with no ansible)
```

Comment:
What if we introduce this virtual env method as it's own sub-section within this section to help users understand that a virtual env is an option?

Reply:
Sounds like a plan, let me do it.

Thank you!
Waldirio

Reply:
@ehelms please, let me know what you think.

Thank you!
Waldirio